src/compiler/rust/canister/src/chunk.rs

```diff
@@ -0,0 +1,6 @@
+pub async fn chunk() {
+    let id = ic_cdk::id();
+    let method = "_azle_chunk";
+    let args_raw = [68, 73, 68, 76, 0, 0]; // '()' pre encoded
```

Comment:
I'm trying to remember...this can't just be left an empty array or an empty tuple? Seems strange

Reply:
I was following the example you wrote in chunk.ts
```typescript
await ic.callRaw(ic.id(), '_azle_chunk', ic.candidEncode('()'), 0n);
```

Reply:
Let's just try to make sure